cli/tools/bench.rs

```diff
@@ -0,0 +1,541 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+use crate::colors;
+use crate::compat;
+use crate::create_main_worker;
+use crate::emit;
+// use crate::file_fetcher::File;
+// use crate::file_watcher;
+// use crate::file_watcher::ResolutionResult;
+use crate::flags::BenchFlags;
+use crate::flags::Flags;
+use crate::fs_util::collect_specifiers;
+use crate::fs_util::is_supported_bench_path;
+use crate::located_script_name;
+use crate::ops;
+use crate::proc_state::ProcState;
+
+use deno_core::error::generic_error;
+use deno_core::error::AnyError;
+use deno_core::futures::future;
+use deno_core::futures::stream;
+use deno_core::futures::StreamExt;
+use deno_core::serde_json::json;
+use deno_core::ModuleSpecifier;
+use deno_graph::ModuleKind;
+use deno_runtime::permissions::Permissions;
+use deno_runtime::tokio_util::run_basic;
+use log::Level;
+use num_format::Locale;
+use num_format::ToFormattedString;
+use serde::Deserialize;
+use std::io::Write;
+use std::path::PathBuf;
+use std::sync::mpsc::channel;
```

Comment:
Maybe use a tokio channel instead? Or is that not possible with ops?

Reply:
This is used across threads, I'm not sure if tokio channel would work in such case.